nixos/modules/services/security/fail2ban.nix

```diff
@@ -274,26 +274,16 @@ in
       "fail2ban/filter.d".source = "${cfg.package}/etc/fail2ban/filter.d/*.conf";
     };
 
+    systemd.packages = [ cfg.package ];
     systemd.services.fail2ban = {
-      description = "Fail2ban Intrusion Prevention System";
-
       wantedBy = [ "multi-user.target" ];
```

Comment:
Thats also in the service, not sure if we need to repeat it here

Reply:
We need it. I tried omitting it, but the wantedBy here enables the service.